src/commands/install.yml

```diff
@@ -0,0 +1,53 @@
+description: Install the Rainforest QA CLI
+parameters:
+  install-path:
+    description: Where to install the CLI
+    type: string
+    default: /usr/local/bin
+  channel:
+    description: The version of the CLI to install
+    type: enum
+    enum: ["stable", "beta", "dev"]
+    default: stable
+  platform:
+    description: Which platform to install the CLI for
+    type: enum
+    enum: ["darwin", "linux"]
+    default: linux
+  format:
+    description: |
+      What compression format to download. Linux must use tgz. MacOS (Darwin) can choose either tgz or zip.
+    type: enum
+    enum: ["zip", "tgz"]
+    default: tgz
+  architecture:
+    description: 386 (32-bit) or amd64 (64-bit)?
```

Comment:
Why the question mark?

Reply:
These were all originally questions. Will fix.